scikits/odes/sundials/cvode.pyx

```diff
@@ -752,12 +781,6 @@ cdef class CVODE:
                                'verbosity']:
                     raise ValueError("Option '%s' can''t be set runtime." % opt)
```

Comment:
Sure, I'm happy to keep it, it got included because I grabbed the new-api and validate-flags changed, squashed them, and put them either in the bucket of validate-flags or not. I'll create a separate pull request which raises a deprecation warning.